scripts/report_scripts/index.Rmd

```diff
@@ -379,74 +433,56 @@ if (length(unique(approved_var_plot$dates)) > 1) {
 
 The following plots provide an overview of detected single nucleotide
 mutations in different locations and how their relative frequency
-changes over time. Furthermore, mutations showing a significant frequency increase over time are highlighted.  
+changes over time. Furthermore, mutations showing a significant frequency
+increase over time are highlighted.  
 
 ```{r mutation_notation, results='asis', eval=MUTATIONS_FOUND}
 cat(create_html_note(paste(
   "Mutations are noted in the pattern of:",
   "\n _**gene :: protein-sequence mutation : NT-sequence mutation**_\n",
   "Please note that this translation was done for single mutations.",
   "Combinations of single-NT mutations that taken together may lead to ",
-  "a different amino acid are not yet taken into account.",sep = "\n")))
+  "a different amino acid are not yet taken into account.",
+  sep = "\n"
+)))
 ```
 
 
 ```{r title_lm_table, results = 'asis', warning=TRUE, eval=MUTATIONS_FOUND}
-  cat(
-"## Increasing mutations
+cat(
+  "## Increasing mutations\n\n",
+  "To show the dynamic of significantly changing mutations over time a linear ",
+  "regression model was applied to the mutationresults across all samples.",
+  sep = ""
+)
 
-To show the dynamic of significantly changing mutations over time a linear regression model was applied to the mutation
-results across all samples.
-"
-  )
-if(!RUN_MUTATION_REGRESSION) {
-  warning("No mutations or not enough mutations were found to perform a regression analysis.\nThe associated plots are skipped.\n")
+if (!RUN_MUTATION_REGRESSION) {
+  warning(paste(
+    "No mutations or not enough mutations were found to perform a",
+    "regression analysis.\nThe associated plots are skipped.\n"
+  ))
 }
 ```
 
-```{r linear_regression, eval=RUN_MUTATION_REGRESSION}
+
+```{r linear_regression_tables, warning=TRUE, results='asis', eval=RUN_MUTATION_REGRESSION}
 source(params$fun_lm)
+
 mutation_sheet <- params$mutation_sheet
 sigmuts.df <- read.csv(mutation_sheet, header = TRUE) %>%
   na_if("") %>%
   # split gene name of for easier matching
-  mutate_all(funs(str_replace(., '^[^:]*:', '')))
+  mutate(across(everything(), ~ str_replace(.x, "^[^:]*:", "")))
```

Comment:
I'm having a hard time following these changes.  Better to separate pure style fixes from functional changes.